web/menu.jspf

```diff
@@ -93,11 +93,15 @@ org.opensolaris.opengrok.web.Util"
         name="project" multiple="multiple" size="<%=
         Math.min(15, projectsSize) %>"><%
         SortedSet<String> pRequested = cfg.getRequestedProjects();
-        for ( Group group : ph.getGroups() ) {
+        for (Group group : ph.getGroups()) {
             Set<Project> groupProjects = ph.getAllGrouped(group);
-            if ( groupProjects.size() > 0) {
+            if (groupProjects.size() > 0) {
                 %><optgroup label="<%= group.getName() %>"><%
                 for (Project p : groupProjects) {
+                    if (!p.isIndexed()) {
```

Comment:
There can be a situation when `groupProjects.size() > 0` this is true but with this `if` statement no project will be printed. Is that ok?